frontend/src/scenes/AdminPage/UserTabContent.tsx

```diff
@@ -72,9 +73,12 @@ export default function UserTabContent() {
     phoneNumber: user.phoneNumber,
     destinationCountry: user.destinationCountry,
     callTime:
-      user.callTime === undefined || user.callTime === null
+      user.callTime == null
```

Comment:
changed this to == to use the abstract equality operator which should cover both (without strict type).

Let me know if you're uncomfortable with this :)

Reply:
Yes, I would prefer to use `===` in all cases, even if `==` is appropriate here. Even if all devs understood the behaviour of `==` in every single case, it is too visually similar to `===`. I would prefer to use only `===` so that readers of the code can be confident that it is always `===`.